src/extension/content-script/batteries/Twitter.ts

```diff
@@ -99,13 +99,14 @@ function battery(): void {
         //  );
         //});
         // if we find a ⚡ emoji we use the text of the next sibling and try to extract a lnurl
-        if (zapElement) {
+        for (const zapElement of Array.from(zapElements)) {
```

Comment:
Array.from is not needed if we set the target in our tsconfig.json to ES6: `"target": "ES6"`.
This is supported in all modern browsers.

Reply:
let's do that